synk.spotify.csproj

```diff
@@ -5,18 +5,30 @@
     <TargetFramework>net7.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
+    <AzureFunctionsVersion>v4</AzureFunctionsVersion>
   </PropertyGroup>
   
   <ItemGroup>
-    <None Update="appsettings.json">
+    <None Update="host.json">

```

Comment:
What potentially was in `application.json` needs to reside in `host.json` now if it is to be used?

Reply:
Nothing. But there is a local.settings.json that need have the values from appsettings when running locally. In deployed state we probably need to manually set these settings. 

Note: nesting is not supported in local.settings.json, so in order to "fake" an object, use __ to go one level in. 
Example: 
```
"Values": {
  "Spotify__ClientId" : "myclientid", 
  "Spotify__Secret": "mysecret"
}
```
is equivalent to 
```
"Spotify" : {
  "ClientId" : "myclientid",
  "Secret" : "mysecret"
}
```